packages/accordion/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@littlespoon/accordion",
+  "version": "1.0.0",
+  "description": "Little Spoon accordion",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "build:flowtypes": "flowgen lib && rm -rf lib && mv exports lib && yarn build",
+    "clean": "rm -rf coverage exports lib",
+    "lint": "eslint .",
+    "lint:fix": "yarn lint --fix",
+    "lint:tsc": "tsc --noEmit",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@littlespoon/icons": "2.0.1",
```

Update to latest version:

```suggestion
    "@littlespoon/icons": "2.0.2",
```